src/app/_components/MobileNavigation.tsx

```diff
@@ -0,0 +1,151 @@
+'use client'
+
+import { Fragment, useState, Dispatch, SetStateAction } from 'react'
+
+import Link from 'next/link'
+
+import { List, X, Icon as IconType } from '@phosphor-icons/react'
+import clsx from 'clsx'
+import { Route } from 'next'
+
+import { Icon } from '@/components/Icon'
+import { Logo } from '@/components/Logo'
+import { SlideOver } from '@/components/SlideOver'
+import { Social } from '@/components/Social'
+import { baseStyles } from '@/components/TextLink'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [PATHS.EVENTS, PATHS.GRANTS]
+const communityItems = [PATHS.ECOSYSTEM, PATHS.GOVERNANCE]
+
+type MobileLinkProps = {
+  label: string
+  path: Route
+  nested?: boolean
+  setOpen: Dispatch<SetStateAction<boolean>>
+}
+
+function MobileLink({ label, path, nested, setOpen }: MobileLinkProps) {
+  const touchTargetSpacing = 5
+  const touchTargetClass = `px-${touchTargetSpacing}`
+  const touchTargetOffsetClass = `-ml-${touchTargetSpacing}`
+
+  return (
+    <li
+      className={clsx(
+        nested ? 'border-l border-brand-400' : touchTargetOffsetClass,
+      )}
+    >
+      <Link
+        href={path}
+        className={clsx(
+          baseStyles,
+          touchTargetClass,
+          'inline-block px-5 py-2.5',
+        )}
+        onClick={() => setOpen(false)}
+      >
+        {label}
+      </Link>
+    </li>
+  )
+}
+
+type IconButtonProps = {
+  icon: IconType
+  label: string
+  onClick: React.ComponentPropsWithoutRef<'button'>['onClick']
+}
+
+function IconButton({ icon: IconComponent, label, onClick }: IconButtonProps) {
+  return (
+    <button
+      className="-mr-4 flex size-12 items-center justify-center focus:outline-2 focus:outline-brand-100 lg:hidden"
```

Comment:
Is `-mr-4` some sort of dependency?

Reply:
`-mr-4` and `size-12` are related, but I'm not sure there's a clear correlation between them, as in, we can't just multiply by 3 to compute one from the other.

Reply:
Isn't `-mr-4` there because of the touchtarget offset?

Reply:
Related >> Asking Filipa if we can make the open and close button look like buttons [>>](https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1715161658918759)

Reply:
Done -> https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/116/commits/32b5c87f57b5c1e3aad6cb9ce396fe2e9d1c05da